sdk/js/src/service/gateways.js

```diff
@@ -184,6 +186,41 @@ class Gateways {
     // Combine all stream sources to one subscription generator.
     return combineStreams(streams)
   }
+
+  // Gateway Configuration Server.
+
+  async getGlobalConf(gatewayId) {
+
+    const token = this._api._token
```

Comment:
Use https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.9/sdk/js/src/util/token.js
Example: https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.9/sdk/js/src/api/stream/stream.js#L52

Reply:
Do we really need this in the sdk? Can we implement this as a standalone function in the console?

Reply:
Yeah it's a pain but in the future it hopefully will become a documented API endpoint. This way it's easier to find and fix at that point than if it's some dangling function in a component